### ISSUE TITLE
Issues building Android app with the latest RN 0.71

### ISSUE THREAD

**Issue by berdyshev** [oblador/react-native-vector-image#31]
In the latest version of RN 0.71 the `build.gradle` has been changed a lot because of adopting New Architecture and introducing new  React Native Gradle Plugin. And now this package doesn't allow to build the Android app with the following error: 

```
FAILURE: Build failed with an exception.

* Where:
Script '/private/tmp/AwesomeProject/node_modules/react-native-vector-image/strip_svgs.gradle' line: 10

* What went wrong:
Cannot get property 'name' on null object

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 20s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* Where:
Script '/private/tmp/AwesomeProject/node_modules/react-native-vector-image/strip_svgs.gradle' line: 10

* What went wrong:
Cannot get property 'name' on null object

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 20s
```

**Comment by jonra1993** [comment]
I created this patch based on this PR https://github.com/oblador/react-native-vector-image/pull/32

react-native-vector-image+0.3.3.patch

```
diff --git a/node_modules/react-native-vector-image/strip_svgs.gradle b/node_modules/react-native-vector-image/strip_svgs.gradle
index 34f8558..76b18f7 100644
--- a/node_modules/react-native-vector-image/strip_svgs.gradle
+++ b/node_modules/react-native-vector-image/strip_svgs.gradle
@@ -1,20 +1,21 @@
 // This script removes .svg files generated by metro.
 
 gradle.projectsEvaluated {
-    def bundleTasks = tasks.findAll { task -> task.name.startsWith("bundle") && task.name.endsWith("JsAndAssets") && !task.name.contains("Debug") && task.enabled }
     android.applicationVariants.all { def variant ->
         def targetName = variant.name.capitalize()
         def targetPath = variant.dirName
-        def bundleTask = tasks.findByName("bundle${targetName}JsAndAssets")
+        def bundleTask = tasks.findByName("createBundle${targetName}JsAndAssets")
         def mergeResourcesTask = tasks.findByName("merge${targetName}Resources")
-        def cleanUpTask = tasks.create(name: "${bundleTask.name}_VectorImageCleanUp", type: Delete) {
-            description = "strip generated svg assets"
-            group = 'react'
-            delete fileTree("$buildDir/generated/res/react/${targetPath}/raw").matching {
-                    include "*.svg"
-                }
-            enabled(targetName != "Debug" && bundleTask.enabled)
+        if (bundleTask) {
+            def cleanUpTask = tasks.create(name: "${bundleTask.name}_VectorImageCleanUp", type: Delete) {
+                description = "strip generated svg assets"
+                group = 'react'
+                delete fileTree("$buildDir/generated/res/react/${targetPath}/raw").matching {
+                        include "*.svg"
+                    }
+                enabled(targetName != "Debug" && bundleTask.enabled)
+            }
+            mergeResourcesTask.dependsOn(cleanUpTask)
         }
-        mergeResourcesTask.dependsOn(cleanUpTask)
     }
 }
\ No newline at end of file

```

**Comment by oblador** [comment]
Fixed in 0.4.0
